### PR TITLE
Enhancing Design with CSS: Fixes grammar

### PR DESCRIPTION
**Pull request by chrisbarbas**
- [x] You have read [The Odin Projects Contributing Guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md).
 - [x] The title summarizes the change and where it happened, for example: "Fixes punctuation in Clean Code lesson".
 - [x] If the PR is related to an open issue, use a [relevant keyword](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests) and reference it with the `#` sign and the issue number.
 - [x] You have previewed your markdown (if any) in the [Odin Markdown Preview Tool](https://www.theodinproject.com/lessons/preview)
 - [x] If changes are requested, please make the changes in a timely manner. After you submit the changes, request another review from the maintainer (top right).

#### 1. Describe the changes made and include why they are necessary or important:

Removed 'can' from the sentence "Sometimes their examples could can be more useful."

### CODE REVIEW

File: html_css/intermediate_css/enhancing-design-with-css.md [hunk 1]
```diff
@@ -42,7 +42,7 @@ Check out [`opacity`](https://developer.mozilla.org/en-US/docs/Web/CSS/opacity)
 ### Additional Resources
 Throughout this lesson we have linked primarily to the MDN docs for each property... however there are _many_ useful reference sites across the web that can be useful and that may show up when searching these items.
 
-* [CSS Tricks](https://css-tricks.com/almanac/properties) has some really great content. Some of it feels less formal and official than the MDN docs, but that means they can be easier to digest.  Sometimes their examples could can be more useful. For example, check out their pages on the [background shorthand](https://css-tricks.com/almanac/properties/b/background/), or [overflow](https://css-tricks.com/almanac/properties/o/overflow).
+* [CSS Tricks](https://css-tricks.com/almanac/properties) has some really great content. Some of it feels less formal and official than the MDN docs, but that means they can be easier to digest.  Sometimes their examples could be more useful. For example, check out their pages on the [background shorthand](https://css-tricks.com/almanac/properties/b/background/), or [overflow](https://css-tricks.com/almanac/properties/o/overflow).
```
Comment (thatblindgeye) [inline html_css/intermediate_css/enhancing-design-with-css.md:45]: I think `could` is the word that should be removed here as it sounds like the text is saynig the CSS Tricks article *can* be more useful than MDN sometimes.

```suggestion
* [CSS Tricks](https://css-tricks.com/almanac/properties) has some really great content. Some of it feels less formal and official than the MDN docs, but that means they can be easier to digest.  Sometimes their examples can be more useful. For example, check out their pages on the [background shorthand](https://css-tricks.com/almanac/properties/b/background/), or [overflow](https://css-tricks.com/almanac/properties/o/overflow).
```

**Review by thatblindgeye — CHANGES_REQUESTED** [review]

**Comment by thatblindgeye** [comment]
Additionally, @chrisbarbas, please re-read the PR template and ensure you've followed it correctly (specifically the title of the PR)

File: html_css/intermediate_css/enhancing-design-with-css.md [hunk 2]
```diff
@@ -53,4 +53,4 @@ This section contains questions for you to check your understanding of this less
 * Which property would you use to add scrollbars to an element?
 * Which property would you use to add a shadow behind an element?
 * Which property would you use to create rounded corners on an element?
-* How would you use border-radius to make a circular element?
\ No newline at end of file
+* How would you use border-radius to make a circular element?
```
Comment (thatblindgeye) [inline html_css/intermediate_css/enhancing-design-with-css.md:56]: Should be the last thing. I believe we need a new line at the end of the lesson files.

**Review by thatblindgeye — CHANGES_REQUESTED** [review]
Last thing it looks like. Thank you for making the update!

**Comment by thatblindgeye** [comment]
@chrisbarbas just wanted to let you know I updated the title of this PR

**Comment by 01zulfi** [comment]
@chrisbarbas Are you still available to make the requested changes? Following inactivity over the next few days, this pull request will be closed and an issue will be put up for anyone to make these changes.

**Comment by zachmmeyer** [comment]
Closed due to being stale.

**Pull request closed without merging**